features/networking/egress-ingress.feature

```diff
@@ -662,3 +662,47 @@ Feature: Egress-ingress related networking scenarios
     When I execute on the pod:
       | curl | --connect-timeout | 5 | --head | www.chsi.com.cn |
     Then the step should fail
+
+
+  # @author jechen@redhat.com
+  # @case_id OCP-44940
+  @admin
```

Comment:
`@network-ovnkubernetes` is also needed for filter the cases. see 
https://github.com/openshift/verification-tests/pull/2427/files#r740800357